### ISSUE TITLE
BUG:  Force plot JS Not Initialized 

### ISSUE THREAD

**Issue by Clovinlee** [shap/shap#3431]
### Issue Description

Javascript Not Initialized error occurs whenever I tries to use the SHAP force plot.
It works fine with other visualization though.

I even used the example code from SHAP github page.

### Minimal Reproducible Example

```python
import xgboost
import shap

# train an XGBoost model
X, y = shap.datasets.california()
model = xgboost.XGBRegressor().fit(X, y)

# explain the model's predictions using SHAP
# (same syntax works for LightGBM, CatBoost, scikit-learn, transformers, Spark, etc.)
explainer = shap.Explainer(model)
shap_values = explainer(X)

# visualize the first prediction's explanation
shap.initjs()
shap.plots.force(shap_values[0])
```

### Expected Behavior

The plot showing normally

### Bug report checklist

- [X] I have checked that this issue has not already been reported.
- [X] I have confirmed this bug exists on the [latest release](https://github.com/shap/shap/releases) of shap.
- [ ] I have confirmed this bug exists on the [master branch](https://github.com/shap/shap/blob/master/CONTRIBUTING.md#installing-from-the-master-branch) of shap.
- [ ] I'd be interested in making a PR to fix this bug

### Installed Versions

latest right now : 0.44.0

**Comment by yacin-h** [comment]
Hello, I encountered the same problem when I ran it on Google Colab. It works fine with shap==0.43.0 though.

**Comment by Clovinlee** [comment]
> Hello, I encountered the same problem when I ran it on Google Colab. It works fine with shap==0.43.0 though.

Yes, fortunately it only happen on 0.44 
Downgrading the version to 0.43 works fine

**Comment by richarddli** [comment]
See https://github.com/shap/shap/issues/3448 for the explanation and fix.

**Comment by connortann** [comment]
This should be fixed in the latest release `0.44.1` which has been published to PyPI and will shortly be on conda-forge. Please let us know if you still encounter the issue.
